### PR TITLE
Add to the note about Azure AD permission recommendations

### PR DESCRIPTION
**Pull request by maxsokolovsky**

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md [hunk 1]
```diff
@@ -113,7 +113,21 @@ Ensure that you set Application permissions, and *not* Delegated permissions. Ot
 
 :::note
 
-Rancher doesn't validate the permissions you grant to the app in Azure. We only support the use of the `Directory.Read.All` application permission.
+Rancher doesn't validate the permissions you grant to the app in Azure. This means you are free to try any permissions you want, just ensure that they are sufficient for Rancher to work with AD users and groups.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md:116]:
```suggestion
Rancher doesn't validate the permissions you grant to the app in Azure.  You're free to try any permissions you want, as long as they allow Rancher to work with AD users and groups.
```

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md [hunk 2]
```diff
@@ -113,7 +113,21 @@ Ensure that you set Application permissions, and *not* Delegated permissions. Ot
...
+
+Specifically, Rancher needs to do the following:
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md:118]:
```suggestion
Specifically, Rancher needs permissions that allow the following actions:
```

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md [hunk 3]
```diff
@@ -113,7 +113,21 @@ Ensure that you set Application permissions, and *not* Delegated permissions. Ot
...
+- Get a user
+- List all users
+- List groups of which a given user is a member
+- Get a group
+- List all groups
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md:123]:
```suggestion
- Get a user.
- List all users.
- List groups of which a given user is a member.
- Get a group.
- List all groups.
```

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md [hunk 4]
```diff
@@ -113,7 +113,21 @@ Ensure that you set Application permissions, and *not* Delegated permissions. Ot
...
+
+Rancher performs these actions either to log a user in or run user/group search. Keep in mind that the permissions must be of type Application.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md:125]:
```suggestion
Rancher performs these actions either to log in a user or to run a user/group search. Keep in mind that the permissions must be of type `Application`.
```

**Review by martyav — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md (hunk 3 above)
Comment (maxsokolovsky) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md:123]: Out of curiosity, why do we need periods at ends of list items? These aren't sentences...
Also, should I add them below where I list example permission combinations?

Comment (martyav) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md]: The style guide we use specifies using a period at the end of each list item if at least one of the list items can be read as a sentence, even if they're very short.

**Review by martyav — APPROVED** [review]

**Pull request merged**